scripts/generateSJCL.js

```diff
@@ -10,7 +10,11 @@ async function init () {
         return
     }
 
-    await exec('cd node_modules/sjcl/ && ./configure --no-export --compress=none --without-all --with-hmac --with-codecHex && make')
```

Comment:
I'm going to guess the perl side works for linux/mac too maybe they can be completely shared.